docs/root/configuration/listeners/network_filters/_include/zookeeper-proxy-filter.yaml

```diff
@@ -0,0 +1,38 @@
+static_resources:
+  listeners:
+    - name: main
+      address:
+        socket_address:
+          address: 127.0.0.1 # Host that zookeeper clients should connect to.
+          port_value: 10001  # Port that zookeeper clients should connect to.
+      filter_chains:
+      - filters:
+        - name: envoy.filters.network.zookeeper_proxy
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.filters.network.zookeeper_proxy.v3.ZooKeeperProxy
+            stat_prefix: zookeeper
+        - name: envoy.filters.network.tcp_proxy
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
+            stat_prefix: tcp
+            cluster: local_zk_server
+  clusters:
+    - name: local_zk_server
+      connect_timeout: 120s
+      type: LOGICAL_DNS
+      lb_policy: ROUND_ROBIN
+      typed_extension_protocol_options:
+        envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+          "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
+          explicit_http_config:
+            http2_protocol_options: {}
+      load_assignment:
+        cluster_name: local_zk_server
+        endpoints:
+          - lb_endpoints:
+              - endpoint:
+                  address:
+                    socket_address:
+                      address: 127.0.0.1
+                      port_value: 2181
+                                           
```

Comment:
new line - not blank line - you probably want your editor to do this automatically

Reply:
thanks, I configured in my vscode ide just now.